inst/tests/tests.Rraw

```diff
@@ -16163,16 +16161,15 @@ test(2097.3, DT[J(NA_integer_)], error="Factor columns must join to factor or ch
 # Fixes: 'Column 2 is length 3 which differs' and 'argument x is missing with no default'
 DT = data.table(id=INT(3,5,2,4,2), rating=LETTERS[5:1])
 groups = c("id", "rating")
-test(2098.1, DT[do.call(order, mget(groups)), verbose=TRUE], ans<-data.table(id=INT(2,2,3,4,5), rating=c("A","C","E","B","D")),
-             output=out<-"forder.c received 5 rows and 2 columns")
-test(2098.2, DT[with(DT, do.call(order, mget(groups))), verbose=TRUE],  ans, output=out)
-test(2098.3, DT[do.call(forder, mget(groups)), verbose=TRUE],           ans, output=out)
-test(2098.4, DT[with(DT, do.call(forder, mget(groups))), verbose=TRUE], ans, output=out)
+test(2098.1, DT[do.call(order, mget(groups)), verbose=TRUE], ans<-data.table(id=INT(2,2,3,4,5), rating=c("A","C","E","B","D")))
+test(2098.2, DT[with(DT, do.call(order, mget(groups))), verbose=TRUE],  ans)
+test(2098.3, DT[do.call(forder, mget(groups)), verbose=TRUE],           ans)
+test(2098.4, DT[with(DT, do.call(forder, mget(groups))), verbose=TRUE], ans)
 old = options(datatable.optimize=0L)
-test(2098.5, DT[do.call(order, mget(groups)), verbose=TRUE],            ans, notOutput="forder.c")
-test(2098.6, DT[with(DT, do.call(order, mget(groups))), verbose=TRUE],  ans, notOutput="forder.c")
-test(2098.7, DT[do.call(forder, mget(groups)), verbose=TRUE],           ans, output=out)
-test(2098.8, DT[with(DT, do.call(forder, mget(groups))), verbose=TRUE], ans, output=out)
+test(2098.5, DT[do.call(order, mget(groups)), verbose=TRUE],            ans)
+test(2098.6, DT[with(DT, do.call(order, mget(groups))), verbose=TRUE],  ans)
+test(2098.7, DT[do.call(forder, mget(groups)), verbose=TRUE],           ans)
+test(2098.8, DT[with(DT, do.call(forder, mget(groups))), verbose=TRUE], ans)
```

Comment:
Include a brief comment explaining the differences among the tests

Reply:
what type of difference are we talking about ?, these test were already there and i just modified them because their output changed due to this pr, Should i leave a comment informing this change??

Reply:
Yes, a comment describing that change and/or what these set of tests do might be good to have.